src/pages/NewChairConsole.js

```diff
@@ -27,12 +27,11 @@ export default function NewChairConsole(props) {
   useEffect(() => {
     const init = async() => {
       try {
-        const [collections, datasets] = await Promise.all([
-          Collections.getCollectionsByRoleName('chairperson'),
-          User.getUserRelevantDatasets()
+        const [collections] = await Promise.all([
+          Collections.getCollectionsByRoleName('chairperson')
         ]);
         setCollections(collections);
-        setRelevantDatasets(datasets);
+        setRelevantDatasets(collections);
```

Comment:
We shouldn't be setting the `relevantDatasets` to the collections, but the fact that this doesn't seem to mess up the `DarCollectionTable` makes me wonder if we still need the `relevantDatasets` state variable in the `NewChairConsole` component at all. I think it's only use here is to be passed to the `DarCollectionTable`, but the `DarCollectionTable` seems to replace it with `collection.datasets` in `processCollectionRowData` before ever using it